pythondata_cpu_vexriscv/verilog/src/main/scala/vexriscv/GenCoreDefault.scala

```diff
@@ -58,6 +60,8 @@ object GenCoreDefault{
       // ex : -dCacheSize=XXX
       opt[Int]("dCacheSize")     action { (v, c) => c.copy(dCacheSize = v) } text("Set data cache size, 0 mean no cache")
       opt[Boolean]("mulDiv")    action { (v, c) => c.copy(mulDiv = v)   } text("set RV32IM")
+      opt[Boolean]("atomics")    action { (v, c) => c.copy(mulDiv = v)   } text("set RV32I[A]")
```

Comment:
@ilya-epifanov , this looks like a copy-and-paste typo?   Or am I misunderstanding something?